lib/src/value/calculation.dart

```diff
@@ -120,6 +124,226 @@ class SassCalculation extends Value {
     return SassCalculation._("max", args);
   }
 
+  /// Creates a `hypot()` calculation with the given [arguments].
+  ///
+  /// Each argument must be either a [SassNumber], a [SassCalculation], an
+  /// unquoted [SassString], a [CalculationOperation], or a
+  /// [CalculationInterpolation]. It must be passed at least one argument.
+  ///
+  /// This automatically simplifies the calculation, so it may return a
+  /// [SassNumber] rather than a [SassCalculation]. It throws an exception if it
+  /// can determine that the calculation will definitely produce invalid CSS.
+  static Value hypot(Iterable<Object> arguments) {
+    var args = _simplifyArguments(arguments);
+    if (args.isEmpty) {
+      throw ArgumentError("hypot() must have at least one argument.");
+    }
+    _verifyCompatibleNumbers(args);
+
+    var subtotal = 0.0;
+    var index = 0;
+    var first = arguments.first;
+    if (first is! SassNumber) return SassCalculation._("hypot", args);
+    for (var number in arguments) {
+      if (number is! SassNumber || number.hasUnit('%')) {
+        return SassCalculation._("hypot", args);
+      }
+      var value = number.convertValueToMatch(
+          first, "numbers[${index + 1}]", "numbers[1]");
+      subtotal += value * value;
+      index++;
+    }
+    return SassNumber.withUnits(math.sqrt(subtotal),
+        numeratorUnits: first.numeratorUnits,
+        denominatorUnits: first.denominatorUnits);
+  }
+
+  /// Creates a `sqrt()` calculation with the given [argument].
+  ///
+  /// The [argument] must be either a [SassNumber], a [SassCalculation], an
+  /// unquoted [SassString], a [CalculationOperation], or a
+  /// [CalculationInterpolation].
+  ///
+  /// This automatically simplifies the calculation, so it may return a
+  /// [SassNumber] rather than a [SassCalculation]. It throws an exception if it
+  /// can determine that the calculation will definitely produce invalid CSS.
+  static Value sqrt(Object argument) {
+    argument = _simplify(argument);
+    if (argument is! SassNumber || argument.hasUnit('%')) {
+      return SassCalculation._("sqrt", [argument]);
+    }
+    return number_lib.sqrt(argument);
+  }
```

Comment:
I tried creating a helper function for this, but it would still need a `argument is SassNumber` in the caller function, since the verification type is not passed. Should I do the double check in both functions or leave it as it is?